packages/sui-js-compiler/swc-config.js

```diff
@@ -1,37 +1,55 @@
-export default {
-  minify: true,
-  jsc: {
-    parser: {
-      syntax: 'ecmascript',
-      jsx: true,
-      dynamicImport: true,
-      privateMethod: true,
-      functionBind: true,
-      exportDefaultFrom: true,
-      exportNamespaceFrom: true,
-      decorators: true,
-      decoratorsBeforeExport: true,
-      topLevelAwait: true,
-      importMeta: true
-    },
-    transform: {
-      legacyDecorator: true,
-      react: {
-        useBuiltins: true,
-        runtime: 'automatic'
-      }
-    },
-    target: 'es5',
-    loose: true,
-    externalHelpers: true
-  },
-  env: {
-    targets: {
-      ie: '11'
+const DEFAULT_LEGACY_BROWSER_TARGETS = {
+  ie: '11'
+}
+
+const DEFAULT_BROWSER_TARGETS = {
+  chrome: '72',
+  edge: '79',
+  safari: '14',
+  firefox: '80',
+  opera: '60',
+  ios: '14'
+}
```

Comment:
```suggestion
const DEFAULT_BROWSER_TARGETS = {
  chrome: '72',
  edge: '79',
  safari: '14',
  firefox: '80',
  opera: '60',
  ios: '14'
}

const DEFAULT_LEGACY_BROWSER_TARGETS = {
  ...DEFAULT_BROWSER_TARGETS,
  ie: '11'
}
```

would that make sense? or just setting `ie: '11'` is enough?

Reply:
Yes! Just setting ie: '11' should be sufficient. This was the previous SWC configuration and is also the most restrictive.